.github/workflows/deploy-all-deploy.yml

```diff
@@ -116,5 +140,5 @@ jobs:
                   echo "Created ${ENV_FILE} with values:"
                   cat src/cashier_frontend/${ENV_FILE}
 
-                  just build_frontend "$ENVIRONMENT"
-                  dfx deploy cashier_frontend --network ${NETWORK}
+                  just build "$ENVIRONMENT"
+                  just deploy-all "$ENVIRONMENT"
```

Comment:
It is not clear if here we need to pass "IC" or "PRODUCTION". Please clean all scripts to use only one or the other